### PR TITLE
Transifex was not happy with empty lines

### PR DESCRIPTION
**Pull request by lespacedunmatin**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -63,24 +63,19 @@
     "step accounts next": "Browse",
     "myaccounts info": "You can now access your Cozy at anytime by clicking 'Continue'",
     "myaccounts next": "Continue",
+    "sign in": "Sign in",
```
Comment (gregorylegarec) [inline src/locales/en.json]: I think that the local key should be more contextualized, to make it clear that it concerns connexion to konnector services. That was it was previously something like `intent service connect`. What do you think about `konnector sign in` or `konnector service sign in` ?

Comment (lespacedunmatin) [reply src/locales/en.json]: I do not understand why you prefer that, but I'll do the change.

**Comment by lespacedunmatin** [comment]
@gregorylegarec :  8a558e9 solves the issue.

File: src/locales/en.json (hunk 1 above)
Comment (lespacedunmatin) [reply src/locales/en.json]: 8a558e9 solves the issue.

**Review by gregorylegarec — APPROVED** [review]

**Pull request merged**